compiler/rustc_target/src/abi/mod.rs

```diff
@@ -234,6 +234,47 @@ pub enum Endian {
     Big,
 }
 
+impl Endian {
+    pub fn name(&self) -> &str {
+        match self {
+            Self::Little => "little",
+            Self::Big => "big",
+        }
+    }
+
+    pub fn is_big(&self) -> bool {
+        Self::Big == *self
+    }
+
+    pub fn is_little(&self) -> bool {
+        Self::Little == *self
+    }
```

Comment:
Could you remove the trivial methods `is_big` and `is_little` and use comparisons like `e == Endian::Little` instead?

Reply:
We could, but then we have to import the enum type everywhere
while its only usecase is a member of a struct.
With this boring trivial methods, we don't need import the enum type.

Reply:
I think importing it would probably be better, is it a totally different import?

Reply:
How? If it's used to assign value to a variable, I would import them. Here we only check the property of it.